lib/decoders/transform.js

```diff
@@ -1,4 +1,7 @@
-import {types} from '../soql/mapper';
+import {
```

Comment:
Is there a difference between this syntax with the `import {...} from ...` and the syntax below: `import ... from ...`?